### DISCUSSION TITLE
Access Logs through UI method

### DISCUSSION THREAD

**Discussion by marius-smg** [FlowiseAI/Flowise#689] [Q&A]
Hey, im struggeling to understand what Log file path i can use when i am on render.com.

**Comment by chungyau97** [comment]
Hi @marius-smg,

Please refer to this doc [render#persistent-disk](https://docs.flowiseai.com/deployment/render#persistent-disk)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/59219b79-a9b7-41da-aa50-3d60fe352d31)

**Reply by marius-smg** [reply]
Thank you! I added this Variable to my Environment, but i dont know how or where to access the log file now.

**Comment by chungyau97** [comment]
Hi @marius-smg,

In the future, we will allow access of logs through UI. cc @HenryHengZJ 

The current workaround for render is use the pay version of it, shown below:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/af5b3512-847a-4a0b-aa18-e5ce1d6722ae)

**Reply by marius-smg** [reply]
Thanks, i have a paid Starter version, i just dont know where i can find the logfiles.

**Reply by chungyau97** [reply]
Hi @marius-smg,

Seem like Render do not provide a way to access folders through UI.

I manage to access the logs path using `Shell` shown below:
Prerequisite: [Setup Persistent Disk](https://docs.flowiseai.com/deployment/render#persistent-disk)
```
cd /opt/render/.flowise/logs
ls -a
cat server.log
```
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/6d290a8c-179d-47be-9cd6-466dfdf1d704)
